calc/circlearea.py

```diff
@@ -0,0 +1,3 @@
+def area(radius):
+
+    return 3.14 * radius ** 2
```

Comment:
Ok, we need to make change here. To calculate circle area, you need PI value.
As its training project, using 3.14 does not harm us. Having said so, proper way is to import proper constant from `math` package.
For example:
at the top of the file do:

`from math import pi`
and then replace `3.14` with pi. Take note that test will probably stop passing